llama_stack/providers/remote/inference/nvidia/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from ._config import NVIDIAConfig
+from ._nvidia import NVIDIAInferenceAdapter
```

Comment:
this should be a dynamic import within `get_adapter_impl()` -- we want configs to be manipulated without needing implementation dependencies.